e2e/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2'
+services:
+
+  broker:
+    image: nats
+    expose:
+      - 4222
+      - 8222
+
+  node:
+    build:
+      context: ..
+      dockerfile: e2e/Dockerfile.node
+    depends_on:
+      - broker
+    privileged: true
+    cap_add:
+      - MKNOD
+      - NET_ADMIN
+    expose:
+      - 1194
+    command: "--data-dir=/mysterium --config-dir=/mysterium/server_config --runtime-dir=/mysterium --broker-address=broker"
+
+  client:
+    build:
+      context: ..
+      #this effectively reuses node image built earlier
+      dockerfile: e2e/Dockerfile.node
+    depends_on:
+      - broker
+      - node
+    privileged: true
+    cap_add:
+      - MKNOD
+      - NET_ADMIN
+    ports:
+      - 4052:4050
+    entrypoint:
+      "/mysterium/mysterium_client"
```

Comment:
We have Dockerfile for client also, just nobody is testing them. Would be nice integration covers existing tools. Because they are breaking from time to time: Dockerfile of server, Dockerfile of client

Reply:
Let's make a clean separation between integration testing and packaging testing